src/plugin/context.rs

```diff
@@ -249,8 +249,8 @@ impl RapierContext {
                             // Update the previous state transforms
                             for (handle, mut interpolation) in interpolation_query.iter_mut() {
                                 if let Some(body) = self.bodies.get(handle.0) {
-                                    interpolation.start = Some(*body.position());
-                                    interpolation.end = None;
+                                    interpolation.start = interpolation.end;
+                                    interpolation.end = Some(*body.position());
```

Comment:
From what I understanded of `bevy_rapier`'s code this change is not valid, because this system execute whole Rapier's subupdates (I stick to name convention from #464), and this code block is executed only for last subupdate. This stores a value from previous subupdate, and next sets final subupdate position as end in lerp system:
https://github.com/dimforge/bevy_rapier/blob/c6bcce4695d596a7a9c8e91748d4dbb3d31f6d13/src/plugin/systems.rs#L554-L556

Maybe in case where you have only a one subupdate a code what are you change is invalid, but I must think about this. But otherwise in my opinion code which is writed earlier is correct.

Reply:
After implementation expected behavior from #464 I do not know how lerp should looks. When delta time of `FixedUpdate` is greater than delta time of `Update`  @liquidev's lerp works good, but in situation when it is lower earlier lerp works good. For equals I do not know this at this time. Someone should investigate this.